samples/boot/oauth2/resource-server/src/main/java/org/springframework/security/samples/config/MessagesApplication.java

```diff
@@ -59,6 +61,11 @@ protected void configure(HttpSecurity http) throws Exception {
 		}
 	}
 
+	@Bean
```

Comment:
Ideally this bean would be added by default for users. It might only be in Boot or it might be in Spring Security config support itself. Can we create a separate ticket in this project to ensure that we do that?

Reply:
Agreed--just one of those things that was in my head, but I should have written down.

Reply:
https://github.com/jzheaux/spring-security-oauth2-resource-server/issues/31